pkg/fsutil/fadvise_linux.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2022 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//go:build linux
+// +build linux
+
+package fsutil
+
+import (
+	"os"
+
+	"golang.org/x/sys/unix"
+)
+
+// DropPageCache us fadvise to tell OS the reclaim memory associated with the file
+// as soon as possible, which can avoid the unexpected I/O latency spike risk.
+func DropPageCache(name string) error {
```

Comment:
Will it return an error if the program runner isn't root?

Reply:
Yes, non-root user can also call `unix.fadvise()`

Reply:
I create a normal user and run the unit test `TestDropPageCache`, it succeeds.